ResidentsSocialCarePlatformApi/V1/Gateways/MosaicGateway.cs

```diff
@@ -170,7 +170,7 @@ private ResidentInformation AttachPhoneNumberToPerson(ResidentInformation person
             return resident;
         }
 
-        private static string GetMostRecentUprn(IEnumerable<Address> addressesForPerson)
+        private static string GetMostRecentUprn(IEnumerable<Infrastructure.Address> addressesForPerson)
```

Comment:
Same question for `Infrastructure` - is it from the auto refactor and do we need it?

Reply:
I think it's doing this to make sure it's referencing the right file for the class since a lot of the classes have the same name and live in different folders. 

But it might be as a result of the auto refactoring that it's done this. We might not need it once we start editing the files.